src/Terminal/index.ts

```diff
@@ -44,6 +45,12 @@ const createServerSettings = (urlVars: { [key: string]: string }): ServerConnect
 
 const main = async (): Promise<void> => {
   const urlVars = getUrlVars();
+
+  // Initialize userContext. Currently only subscrptionId is required by TelemetryProcessor
+  updateUserContext({
+    subscriptionId: urlVars["subscriptionId"]
```

Comment:
Let's use a constant for the "subscriptionId"

Reply:
Replaced all Terminal query params with constant